addons/stock/models/stock_rule.py

```diff
@@ -282,7 +284,7 @@ def _get_stock_move_values(self, product_id, product_qty, product_uom, location_
             'propagate_cancel': self.propagate_cancel,
             'propagate_date': self.propagate_date,
             'propagate_date_minimum_delta': self.propagate_date_minimum_delta,
-            'description_picking': product_id._get_description(self.picking_type_id),
+            'description_picking': values.get('description') and (product_desc + values.get('description')) or product_desc,
```

Comment:
seems very weird, you added the description field but you changed the use of description_picking?

Reply:
done